### PR TITLE
Implement setting video fit state

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch implements setting video fit state for media remote.
And, it adds MediaFitState to define state on the view model.

Refs: #1118

### CODE REVIEW

File: src/org/triple/banana/media/MediaFitState.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Copyright 2021 The Triple Banana Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package org.triple.banana.media;
+
+public enum MediaFitState {
```
Comment (romandev) [inline src/org/triple/banana/media/MediaFitState.java:8]: If you can write some utility method in Enum class, then you can get rid of a lot of switch statements.
```
    enum MediaFitState {
        FILL,
        COVER,
        CONTAIN;

        @Override
        public @NonNull String toString() {
            return super.toString().toLowerCase();
        }

        public static @NonNull MediaFitState from(@NonNull String value) {
            try {
                return MediaFitState.valueOf(value.toUpperCase());
            } catch (Exception e) {
                return CONTAIN;
            }
        }

        public @NonNull MediaFitState toggle() {
            return values()[(ordinal() + 1) % values().length];
        }
    }
```

File: src/org/triple/banana/media_remote/MediaRemoteService.java [hunk 2]
```diff
@@ -189,6 +190,21 @@ public void onMediaRemoteButtonClicked(int id) {
...
+                    break;
+                case COVER:
+                    fitState = MediaFitState.CONTAIN;
+                    break;
+                case CONTAIN:
+                default:
+                    fitState = MediaFitState.FILL;
+                    break;
+            }
+            mMediaController.setVideoFit(fitState);
```
Comment (romandev) [inline src/org/triple/banana/media_remote/MediaRemoteService.java:207]: mMediaController.setVideoFit(mViewModel.getData().getFitState().toggle());

File: src/org/triple/banana/media/MediaCommandProcessor.java [hunk 3]
```diff
@@ -79,4 +80,21 @@ public void setRelativeVolume(float value) {
...
+                break;
+            case COVER:
+                fitState = "cover";
+                break;
+            case CONTAIN:
+            default:
+                fitState = "contain";
+                break;
+        }
+        runMediaCommand("style.objectFit = '" + fitState + "'");
```
Comment (romandev) [inline src/org/triple/banana/media/MediaCommandProcessor.java:98]: runMediaCommand("style.objectFit = '" + state + "'");

File: src/org/triple/banana/media/MediaFitState.java (hunk 1 above)
Comment (jdragonbae) [reply src/org/triple/banana/media/MediaFitState.java:8]: Thanks for the review! Sounds good to me.

File: src/org/triple/banana/media_remote/MediaRemoteService.java (hunk 2 above)
Comment (jdragonbae) [reply src/org/triple/banana/media_remote/MediaRemoteService.java]: Done.

File: src/org/triple/banana/media/MediaCommandProcessor.java (hunk 3 above)
Comment (jdragonbae) [reply src/org/triple/banana/media/MediaCommandProcessor.java]: Done.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**